src/main/resources/application.yml

```diff
@@ -88,7 +88,13 @@ authorities:
         reset:
           enabled: true
           validity: 86400
-
+   webauthn:
+      rpid: ${WEBAUTHN_RP_ID:localhost}
```

Comment:
this should match applicationURL

Reply:
Done in 14510e3d572c17617b4caf1960a7d8a7e2fbd9ad